apps/prairielearn/src/tests/helperDb.ts

```diff
@@ -36,7 +37,18 @@ const postgresTestUtils = sqldb.makePostgresTestUtils({
   },
 });
 
-async function runMigrationsAndSprocs(dbName: string, runMigrations: boolean): Promise<void> {
+async function runMigrationsAndSprocs({
```

Comment:
I made `runMigrations` this union of boolean and an object, I could also do e.g. an `enabled: true` flag in the argument.